instances/latest/terminologies/molecularEntity/GTPDisodiumSalt.jsonld

```diff
@@ -0,0 +1,19 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/molecularEntity/GTPDisodiumSalt",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+  "definition": "Guanosine 5'-triphosphate (GTP) sodium salt hydrate.",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "GTP, Disodium salt",
```

Comment:
I think disodium should not be capitalized.

Reply:
The capitalisation matches the PubChem entry, but I agree.